test/verify/machineslib.py

```diff
@@ -1258,6 +1260,20 @@ def installWithErrorTest(dialog):
                                              os_name=config.NOVELL_NETWARE_6,
                                              start_vm=False))
 
+            m.execute("qemu-img create -f qcow2 /var/lib/libvirt/vmTest20.qcow2 128M")
+            m.execute("virsh pool-refresh libvirt")
+
+            createTest(TestMachines.VmDialog(self, "subVmTestCreate20", sourceType='pxe',
```

Comment:
We don't use names with numbers anymore. Just leave the name field empty.

Reply:
Please address the above comment. The commit which removed the numbering in the tests names is already merged.